IntegrationAdaptersTests/RegisterHospitalTests.cs

```diff
@@ -29,17 +29,51 @@ public void Is_pharmacy_already_existing_on_hospital()
         [Fact]
         public void Is_pharmacy_not_existing_on_hospital()
         {
-            var options = new DbContextOptionsBuilder<HealthCareSystemDbContext>()
-                .UseMySql(connectionString: "server=localhost;port=3306;database=newmydb;user=root;password=root")
-                .Options;
+            ApiService service = new ApiService();
 
-            ApiService service = new ApiService(new HealthCareSystemDbContext(options));
-
-            Api a = new Api("m", "m", "m");
+            Api a = new Api("l", "l", "l");
 
             bool notExists = service.RegisterHospitalOnPharmacy(a);
```

Comment:
A better name for this variable would be registerSuccessful, since the service doesn't know why the hospital couldn't be registered or that api can't be added (the repository contains that logic)